### PR TITLE
ci: add new linters

### PR DESCRIPTION
**Pull request by tklauser**
As suggested by Robin. See individual commits.

### CODE REVIEW

File: pkg/printer/printer.go [hunk 1]
```diff
@@ -644,7 +644,7 @@ func fmtEndpointShort(ep *pb.Endpoint) string {
 	str := fmt.Sprintf("ID: %d", ep.GetID())
 	if ns, pod := ep.GetNamespace(), ep.GetPodName(); ns != "" && pod != "" {
 		str = fmt.Sprintf("%s/%s (%s)", ns, pod, str)
-	} else if lbls := ep.GetLabels(); len(lbls) == 1 && strings.HasPrefix("reserved:", lbls[0]) {
+	} else if lbls := ep.GetLabels(); len(lbls) == 1 && strings.HasPrefix(lbls[0], "reserved:") {
```
Comment (rolinh) [inline pkg/printer/printer.go:647]: This change should probably be backported.

**Review by rolinh — APPROVED** [review]
Awesome :rocket:

File: pkg/printer/printer.go (hunk 1 above)
Comment (tklauser) [reply pkg/printer/printer.go:647]: I'll split it into a separate commit to simplify backporting.

Comment (tklauser) [reply pkg/printer/printer.go:647]: 890e3735f670623fa72146e583d8b80d4b2a3ddc

**Pull request merged**